src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/trust/TrustDataDto.java

```diff
@@ -14,6 +14,7 @@ public class TrustDataDto {
     public static final String TRUST_NAME_FIELD = "trust_name";
     public static final String CREATION_DATE_FIELD = "creation_date";
     public static final String UNABLE_TO_OBTAIN_ALL_TRUST_INFO_FIELD = "unable_to_obtain_all_trust_info";
+    public static final String HISTORICAL_BO_FIELD = "HISTORICAL_BO";
```

Comment:
you should also change the @JsonPropertry("HISTORICAL_BO") to use the new constant HISTORICAL_BO_FIELD

Reply:
thanks @markpit I have corrected it.